framework/src/materials/MaterialPropertyStorage.C

```diff
@@ -355,6 +355,26 @@ MaterialPropertyStorage::copy(MaterialData & material_data,
   }
 }
 
+void
+MaterialPropertyStorage::copy(MaterialData & material_data,
+                              const Elem * elem_to,
+                              const Elem * elem_from,
+                              unsigned int side,
+                              unsigned int n_qpoints)
+{
+  initProps(material_data, *elem_to, side, n_qpoints);
```

Comment:
Not sure how we should handle potential dereferencing errors here...

Reply:
Again - those functions all just need elem pointers - they never actually access the elem instance.

Reply:
Ah you are right. Do you mind if I create another version of initProps that takes in element pointers?